code/modules/mapping/random_rooms.dm

```diff
@@ -1214,4 +1214,40 @@
 	centerspawner = FALSE
 	template_height = 5
 	template_width = 3
-	weight = 2
\ No newline at end of file
+	weight = 2
+
+/datum/map_template/random_room/sk_ren001
```

Comment:
these should be in their own folder

Reply:
do you want a seperate folder for each engine template ?